<?xml version="1.0" encoding="UTF-8" ?>

<launch>
    <!-- MAP accumulator -->
    <node name="map_accumulator" pkg="rpz_planning" type="map_accumulator.py" output="screen">
        <remap from="local_map" to="updated_map"/>
    </node>
</launch>
